driver/config/provider.go

```diff
@@ -186,18 +186,10 @@ func (p *DefaultProvider) IsDevelopmentMode(ctx context.Context) bool {
 }
 
 func (p *DefaultProvider) WellKnownKeys(ctx context.Context, include ...string) []string {
-	if p.AccessTokenStrategy(ctx) == AccessTokenJWTStrategy {
-		include = append(include, x.OAuth2JWTKeyName)
-	}
-
-	include = append(include, x.OpenIDConnectKeyName)
+	include = append(include, x.OAuth2JWTKeyName, x.OpenIDConnectKeyName)
```

Comment:
The `.well-known` endpoint does not depend on a specific client, so we don't know whether or not we need to also list the JWT key names here.

@aeneasr I see no harm in just always including the JWT key names. Correct?

Reply:
Yeah, it's fine!